temporalio/client.py

```diff
@@ -1108,6 +1108,46 @@ async def get_worker_task_reachability(
             )
         )
 
+    async def describe_task_queue(
```

Comment:
Have we considered a top level scaling/tuning API instead that takes task queue types as a filter?  That might be more discoverable for somebody perusing the CLI/API command surface, who knows they want to do scaling but doesn't think of that when they see task queue.  
It would also allow us to add deploy group as a filter once that ships.
And finally, it could allow us to embed other information within the API call such as related namespace-level information, for example if the namespace has rate limits that should also be taken into account.

Reply:
Additional context.  Versioning team originally shipped APIs within task queue, and Max asked us to make them top-level and simply refer to task queue from them.  (Later on, we decided to switch to deploy group, but the top-level API concept remains.)
This feels like it could be analogous.

Reply:
> Have we considered a top level scaling/tuning API instead that takes task queue types as a filter?

I am not sure. This is the Python review for the SDK API that already exists in Go and calls the already existing server API. I think existential questions concerning the API may need to be asked in a place with broader reach.